### ISSUE TITLE
Criar arquivo de serviço para o módulo ClientAdmin

### ISSUE THREAD

**Issue by ricardopoppi** [cidadedemocratica/polisClientAdmin#11]
hj executa pelo arquivo "x"

**Comment by andrebsguedes** [comment]
Esse arquivo deve ser para a inicialização via systemd?

**Comment by ricardopoppi** [comment]
systemd ok, já automatiza pra debian e ubuntu né?

**Comment by andrebsguedes** [comment]
Sim, para as versões mais recentes do Debian e Ubuntu, para quase todas as distribuições baseadas em RedHat e também o ArchLinux.

Essencialmente qualquer distribuição que tenha incorporado o Systemd como inicializador.
